includes/class-main.php

```
@@ -135,21 +143,30 @@ private function includes(): void {
 		require_once WPPO_PLUGIN_PATH . 'includes/class-asset-manager.php';
 		require_once WPPO_PLUGIN_PATH . 'includes/class-htaccess-handler.php';
+		require_once WPPO_PLUGIN_PATH . 'includes/class-server-rules.php';
 		require_once WPPO_PLUGIN_PATH . 'includes/class-core-tweaks.php';
 		require_once WPPO_PLUGIN_PATH . 'includes/class-object-cache.php';
 
 		// Phase 1 — Local Diagnostics (v1.5.0).
-		// Load on admin or AJAX — lazy-load for REST requests in the REST handler.
-		if ( is_admin() || wp_doing_ajax() ) {
+		// Load on admin, AJAX, Cron, or REST API requests.
+		if ( is_admin() || wp_doing_ajax() || wp_doing_cron() || ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) {
 			require_once WPPO_PLUGIN_PATH . 'includes/class-telemetry.php';
```

PHPStan (2.1.51)

[warning] Constant WPPO_PLUGIN_PATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)